test/test_prototype_transforms_functional.py

```diff
@@ -332,6 +332,12 @@ def crop_bounding_box():
         )
 
 
+@register_kernel_info_from_sample_inputs_fn
+def vertical_flip_segmentation_mask():
+    for mask in make_segmentation_masks(extra_dims=((), (4,))):
```

Comment:
@federicopozzi33 can't we use default extra dims ?

Reply:
Yep, done.